### DISCUSSION TITLE
Github Student pack not eligible?

### DISCUSSION THREAD

**Discussion by LeZellus** [community/community#22714] [New to GitHub]
<p>Hello, my name is Mathéo Zeller and I’m french student developer with OpenClassRoom school. Actually I’m running the PHP/symfony course.</p>
<p>Yesterday, I’ve tried to get the student pack but Github turned down my request. I want to know why because my school is recognized by github and I’ve upload a document that proves my schooling.</p>
<p>What I need to do to upgrade my ask ?<br>
Why github deny ?</p>
<p>Thanks for help</p>

**Comment by Mwimwii** [comment]
<p>From my experience, its quite hard to get the github developer pack outside the<br>
US or NA <strong>without</strong> a school email. If you have a <strong>.edu, .ac.jp, etc.</strong> Github verifies those<br>
requests instantly. If not, its most likely handled by the faculty at your school.</p>

**Comment by LeZellus** [comment]
<p>Ok, so what I need to do ? Contact github by email or contact my school to tell them to ask GitHub Student pack ?</p>

**Comment by Mwimwii** [comment]
<p>If you have a <strong>.edu</strong> email address, link it to your github account and try the registerting for the education pack again. If you don’t have an <strong>.edu</strong> email, contact your school, or atleast a lecturer you know. These are programs are usually lead by someone or the whole faculty.</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/zellr">@ZellR</a>,</p>
<p>If you have questions regarding a denied Student Developer Pack application, or other application issues that our <a href="https://github.community/t/github-student-developer-pack-faq/124041/4">FAQ</a> does not address, please reach out to our Education team for assistance via our <a href="https://support.github.com/contact">contact</a> form.</p>
<p>Our Education team will be able to take a look at your application and provide any guidance on appeals, reapplying, or renewals. We’re unable  to review any applications here.</p>
<p>Thanks!</p>

**Comment by Mwimwii** [comment]
<p>Also try and ask some of your classmates if they have access to the education pack.</p>

**Comment by LeZellus** [comment]
<p>My mentor says yes, but isn’t able to help me any more.</p>

**Comment by Mwimwii** [comment]
<p>Well, there are two possibilities:</p>
<ul>
<li>Your mentor received access to the Github Education Pack from their previous institution <strong>.edu</strong> email</li>
</ul>
<p>or</p>
<ul>
<li>They have a special mentor email that allows them to access the pack. (The Pack is also available for teachers)</li>
</ul>
<p>Contacting github to review your application is mandatory, but if you really want to take action, you can gather your classmates together to request access to the pack</p>

**Comment by LeZellus** [comment]
<p>I’ll do something , yes. I’ve already contact support after wordsoup answer. Actually I’m waiting for the response</p>
<p>thanks for help</p>
